Language/Expressions/Constants/Constant_Contexts/constant_context_A01_t01.dart

```diff
@@ -0,0 +1,29 @@
+// Copyright (c) 2024, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+/// @assertion Let e be an expression; e occurs in a constant context iff one of
+/// the following applies:
+/// • e is an element of a list or set literal whose first token is const, or e
+///   is a key or a value of an entry of a map literal whose first token is
+///   const.
+///
+/// @description Checks that an expression `e` is a constant expression if `e`
+/// is an element of a list whose first token is `const`. In this case `const`
```

Comment:
I think we should say 'a list literal', because that's a syntactic construct, a plain set could be the result of computations with basically _any_ syntax
```suggestion
/// is an element of a list literal whose first token is `const`. In this case the `const`
```
Same thing for 'map' in the next test.